orca_detector/live_feed_listener.py

```diff
@@ -42,18 +42,20 @@ def _save_audio_segments(stream_url,
     The result 
     """
 
-    file_name = f"{stream_name}_%02d.wav"
-    output_file = os.path.join(output_path, file_name)
-
     mix_with_command = ''
+    mix_with_name = ''
     if os.path.exists(mix_with):
         print(f'Mixing with {mix_with}')
+        mix_with_name = '_' + os.path.split(mix_with)[1].replace('.wav', '')
```

Comment:
is the filename guaranteed to be in lowercase?

Reply:
good point.  I just pushed a commit to lower() first.